codec-http/src/main/java/io/netty/handler/codec/http/ClientCookieEncoder.java

```diff
@@ -15,6 +15,8 @@
  */
 package io.netty.handler.codec.http;
 
+import io.netty.handler.codec.http.cookie.ClientCookieDecoder;
+
```

Comment:
IMHO, it's not a good thing to add an import for the sheer sake of javadoc while the actual code doesn't use it. I would rather use the fully qualified class name in javadoc then.